nixos/tests/languagetool.nix

```diff
@@ -0,0 +1,25 @@
+import ./make-test-python.nix ({ pkgs, lib, ... }:
+let port = 8082;
+in {
+  name = "languagetool";
+  meta = with lib.maintainers; { maintainers = [ fbeffa ]; };
+
+  nodes.machine = { ... }:
+    {
+      services.languagetool.enable = true;
+      services.languagetool.port = port;
+    };
+
+  testScript = ''
+    def is_up(_) -> bool:
+        status, _ = machine.execute('curl -d "language=en-US" -d "text=a simple test" http://localhost:${toString port}/v2/check')
+        return status == 0
```

Comment:
why not `wait_until_succeeds` where this is used?

Reply:
I just copied the test for another service. I've changed it.